src/GitVersionCore/VersionCalculation/NextVersionCalculator.cs

```diff
@@ -29,7 +29,7 @@ public class NextVersionCalculator : INextVersionCalculator
 
         public SemanticVersion FindVersion()
         {
-            log.Info($"Running against branch: {context.CurrentBranch.FriendlyName} ({(context.CurrentCommit == null ? "-" : context.CurrentCommit.Sha)})");
+            log.Info($"Running against branch: {context.CurrentBranch} ({(context.CurrentCommit != null ? context.CurrentCommit.ToString() : "-")})");
```

Comment:
This should work, no?

```suggestion
            log.Info($"Running against branch: {context.CurrentBranch} ({(context.CurrentCommit?.ToString() ?? "-")})");
```

Reply:
makes sense, will change